zerver/webhooks/dialogflow/doc.md

```diff
@@ -0,0 +1,34 @@
+Get personal message notifications in Zulip for the results of your Dialogflow queries!
+
+Next, on your {{ settings_html|safe }}, [create a bot](/help/add-a-bot-or-integration) for
+{{ integration_display_name }}. Make sure that you select
+**Incoming webhook** as the **Bot type**:
+
+![](/static/images/help/bot_types.png)
+
+The API key for an incoming webhook bot cannot be used to read messages out
+of Zulip. Thus, using an incoming webhook bot lowers the security risk of
+exposing the bot's API key to a third-party service.
+
+Construct the URL for the Dialogflow bot using the bot's API key and your Zulip email.
+
+The webhook URL should look like:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
+Modify the parameters of the URL above where `api_key` is the API key of your Zulip bot
+and `email` is your Zulip email.
+
+* Go to the **Fulfillment** settings of your Dialogflow app and enable **Webhooks**.
+Provide the URL constructed above that the notifications should be sent to.
+
+![](/static/images/integrations/dialogflow/001.png)
+
+* Then, go to **Intents** and at the bottom, check the box **Use webhook**
+under **Fulfillment**.
+
+![](/static/images/integrations/dialogflow/002.png)
```

Comment:
Two things:
* Remove all the screenshots except the one at the end for the sample message, we don't need them and we are gonna remove all screenshots from our webhook docs at some point.
* Combine the last two points into one numbered step.